docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
   ufabc-next-db:
     image: mongo:latest
     env_file:
-      - .env.local
+      - .env
     container_name: $MONGODB_NAME
     restart: always
```

Comment:
nit: restart always has a weird functionality, it creates the container every time you boot up your system instead of when you run docker-compose up, which can be annoying because docker-compose down doesn't work, so you have to docker stop then docker rm the container if you wanna reboot it or use the external port in another container. I believe we should remove it

Reply:
never knew about that, will commit with the changes